app/src/js/components/AsyncCommands/AsyncCommands.js

```diff
@@ -134,6 +137,18 @@ class AsyncCommand extends React.Component {
               </Modal.Footer>
             </Modal>
           ) : null }
+          { (activeModal && confirmModal) ? (
+            <DefaultModal
+              className='link--no-underline'
+              onCancel={this.close}
+              onCloseModal={this.cancel}
+              cancelButtonText={'Close'}
+              hasConfirmButton={false}
+              title={text}
+              children={postActionText}
+              showModal={activeModal}
+              cancelButtonClass={'button--cancel'}
+            />) : null }
```

Comment:
this type of statement can be made more concise with 
```         
 { (activeModal && confirmModal) && (
            <DefaultModal
              className='link--no-underline'
              onCancel={this.close}
              onCloseModal={this.cancel}
              cancelButtonText={'Close'}
              hasConfirmButton={false}
              title={text}
              children={postActionText}
              showModal={activeModal}
              cancelButtonClass={'button--cancel'}
            />) }
```
With the `&&`, only when the conditional passes will the modal render